channel/index.d.ts

```diff
@@ -0,0 +1,22 @@
+import {User} from "../user";
```

Comment:
your file has to be ``index.d.ts``

Reply:
Isn't the file name index.d.ts and I will do the linting